Guide-to-migrate-e2e-tests.md

```diff
@@ -0,0 +1,136 @@
+## Table of contents
+
+* [Overview](#overview)
+* [Selectors](#selectors)
+* [Events on selected elements](#events-on-selected-elements)
+* [Points to note while migrating](#points-to-note-while-migrating)
+* [Example migration](#example-migration)
+
+## Overview
+
+Currently, Oppia End-to-End tests are in a hybrid state where we have both WebdriverIO and Protractor test suites. The GSoC project **Migrate away from Protractor** aims to migrate all the existing End-to-End tests to WebdriverIO.
+
+This guide is to help contributors in migrating end-to-end tests from Protractor to WebdriverIO during the hybrid state.
+
+## Selectors
+
+A selector is used to query an element. The WebDriver Protocol provides several selector strategies to query an element. WebdriverIO simplifies them to keep selecting elements simple
+
+The `$` command is a short way to call the findElement command in order to fetch a single element on the page
+
+| Goal                                     | Protractor                               |WebdriverIO           |
+| -----------------------------------------| -----------------------------------------|----------------------|
+| Select element by class name             |`element(by.css('.some-css-class'))`           |`$('.some-css-class')`     |
+| Select element containing certain string Text |`element(by.cssContainingText(tag, text))`|`$('tag=text')`       |
+| Selecting Multiple Elements              |`element.all(by.css('.some-css-class'))`       |`$$('..some-css-class')`    |
+| Select element by ID                     |`element(by.css('#some-css-id'))`              |`$('#some-css-id')`        |
+| Select element inside a element with some class name |`element(by.css('some-css-class')).element(by.css('tag-within-css'))`  | `$('some-css-class').$('tag-within-css’)`      |
+| Selecting element one by one in loop     |`element.get(i)`                          |`$$(selector)[i]`     |
+
+There is no proper substitute available for the command `element.getWebElement()` in webdriverIO, but we can use `browser.findElement()` as a substitute for this.
+
+For example, take look at the following code of protractor:
+
+```js
+ await browser.executeScript(
+    'arguments[0].click()', await clickableElement.getWebElement());
```

Comment:
Indentation still not fixed ([original comment](https://github.com/oppia/oppia-web-developer-docs/pull/7#discussion_r900415005)). Also, `await` is indented unnecessarily by one space, which will annoy anyone trying to copy-paste this.

Ditto elsewhere

Reply:
Forgot to update it here, done now.